### PR TITLE
added the ability to add comments to articles

### PR DESCRIPTION
**Pull request by AndrewRPerez**
#### What's this PR do?

Adds the ability to add comments to articles.
#### Where should the reviewer start?

  spec/features/adding_comments_spec.rb
#### How should this be manually tested?

View an article and fill in the forms for the comment, then click the 'comment' button.
#### What are the relevant tickets?

N/A
#### Screenshots (if appropriate)

image
#### Definition of Done:
- [x] Is there appropriate test coverage?
- [x] Does this PR have any migrations?
- [N] Does this add new dependencies?

### CODE REVIEW

**Comment by AndrewRPerez** [comment]
@marshallds would you review this?

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def create
+    @article = Article.find(params[:article_id])
```
Comment (sbaus42) [inline app/controllers/comments_controller.rb:3]: Should strong params be used throughout the whole method if they're being used at all?

File: app/views/comments/_comment.html.slim [hunk 2]
```diff
@@ -0,0 +1,11 @@
+hr
+li
+  article.uk-comment
+    header.uk-comment-header
+      h4.uk-comment-title= comment.subject
+      ul.uk-comment-meta.uk-subnav.uk-subnav-line
+        li= comment.name
+        li= link_to 'reply', url_for
```
Comment (fleaplus) [inline app/views/comments/_comment.html.slim:8]: Does the url_for method need any params here?

**Comment by sbaus42** [comment]
:+1:

File: spec/features/add_comments_spec.rb [hunk 3]
```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+describe 'Adding comments' do
+  let(:article) { create :article }
+  let(:comment_attributes) { attributes_for :comment }
+
```
Comment (fleaplus) [inline spec/features/add_comments_spec.rb:6]: You could add a before block here with the call to `visit blog_path article`

**Pull request merged**